qi-lib/private/util.rkt

```diff
@@ -1,5 +1,6 @@
 #lang racket/base
 
+(provide (for-syntax params-parser))
```

Comment:
nit: this could be included in the `provide` form below as a sub-form `(provide ... (for-syntax params-parser))` instead of as a separate form